robot/web/stream/server.go

```diff
@@ -73,70 +85,83 @@ func (e *StreamAlreadyRegisteredError) Error() string {
 }
 
 // NewStream informs the stream server of new streams that are capable of being streamed.
-func (ss *Server) NewStream(config gostream.StreamConfig) (gostream.Stream, error) {
-	ss.mu.Lock()
-	defer ss.mu.Unlock()
+func (server *Server) NewStream(config gostream.StreamConfig) (gostream.Stream, error) {
+	server.mu.Lock()
+	defer server.mu.Unlock()
 
-	if _, ok := ss.nameToStreamState[config.Name]; ok {
+	if _, ok := server.nameToStreamState[config.Name]; ok {
 		return nil, &StreamAlreadyRegisteredError{config.Name}
 	}
 
-	stream, err := gostream.NewStream(config, ss.logger)
+	stream, err := gostream.NewStream(config, server.logger)
 	if err != nil {
 		return nil, err
 	}
 
-	if err = ss.add(stream); err != nil {
+	if err = server.add(stream); err != nil {
 		return nil, err
 	}
 
 	return stream, nil
 }
 
 // ListStreams implements part of the StreamServiceServer.
-func (ss *Server) ListStreams(ctx context.Context, req *streampb.ListStreamsRequest) (*streampb.ListStreamsResponse, error) {
+func (server *Server) ListStreams(ctx context.Context, req *streampb.ListStreamsRequest) (*streampb.ListStreamsResponse, error) {
 	_, span := trace.StartSpan(ctx, "stream::server::ListStreams")
 	defer span.End()
-	ss.mu.RLock()
-	defer ss.mu.RUnlock()
+	server.mu.RLock()
+	defer server.mu.RUnlock()
 
-	names := make([]string, 0, len(ss.streamNames))
-	for _, name := range ss.streamNames {
-		streamState := ss.nameToStreamState[name]
-		names = append(names, streamState.Stream.Name())
+	names := make([]string, 0, len(server.nameToStreamState))
+	for name := range server.nameToStreamState {
+		names = append(names, name)
 	}
 	return &streampb.ListStreamsResponse{Names: names}, nil
 }
 
 // AddStream implements part of the StreamServiceServer.
-func (ss *Server) AddStream(ctx context.Context, req *streampb.AddStreamRequest) (*streampb.AddStreamResponse, error) {
+func (server *Server) AddStream(ctx context.Context, req *streampb.AddStreamRequest) (*streampb.AddStreamResponse, error) {
 	ctx, span := trace.StartSpan(ctx, "stream::server::AddStream")
 	defer span.End()
-	// Get the peer connection
+	// Get the peer connection to the caller.
 	pc, ok := rpc.ContextPeerConnection(ctx)
+	server.logger.Infow("Adding video stream", "name", req.Name, "peerConn", pc)
+	defer server.logger.Warnf("AddStream END %s", req.Name)
```

Comment:
Do we want to keep this at warn or downgrade this to Info? 

Users may be surprised to see warning logs whenever they open a video stream.